starter-code/src/StackDataStructure.js

```diff
@@ -1,2 +1,39 @@
-function StackDataStructure () {
+function StackDataStructure() {
+  this.stackControl = []
+  this.MAX_SIZE = 10
+};
+
+StackDataStructure.prototype.isEmpty = function() {
+  if (this.stackControl.length == 0) {
+    return true;
+  } else {
+    return false;
+  }
+};
+
```

Comment:
Clementina, está función está bien, pero con la condición te basta para que te devuelva true /false. Te serviría con poner únicamente:

StackDataStructure.prototype.isEmpty = function () {
  return this.stackControl.length === 0;
};